shotgun_api3/shotgun.py

```diff
@@ -76,6 +89,22 @@
 LOG.setLevel(logging.WARN)
 
 
+class OrderItem(TypedDict):
+    field_name: str
+    direction: str
+
+
+class GroupingItem(TypedDict):
+    field: str
+    type: str
+    direction: str
+
+
+class BaseEntity(TypedDict, total=False):
+    id: int
+    type: str
```

Comment:
I added some `TypedDicts` to try to fill in some of the required dictionary fields.  Let me know if you see anything wrong here.